webpack.mix.js

```diff
@@ -0,0 +1,86 @@
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const mix = require('laravel-mix');
+const { escapeRegExp } = require('lodash');
```

Comment:
I know it's not that important in `webpack.mix.js`, but you should avoid loading lodash completly otherwise tree shaking won't work (i.e. on files compiled by Webback). I also like to prefix those functions with an underscore to make it obvious it came from loadash. Use this instead:

```suggestion
const _escapeRegExp = require('lodash/escapeRegExp');
```

Reply:
The suggestion was implemented.